build-script.sh

```diff
@@ -1,34 +1,37 @@
 #!/usr/bin/bash
-set -x
 
-organization=test
 prefix=perf-test
-num_users=1000
-num_tags=100
-
+num_users=${NUM_USERS:-10}
+num_images=${NUM_IMAGES:-10}
+num_tags=${NUM_TAGS:-10}
+user_prefix=${NUM_TAGS:-perf-test}
+quay=${QUAY_DOMAIN}
+token=${TOKEN}
 pick=$((1 + RANDOM % 1000))
```

Comment:
this should be `1 + RANDOM % ${num_users}`... Because we could of created only 10 users but from this we could get `perf-test_user_500`.